### DISCUSSION TITLE
Transfer subdomains / Share domain ownership

### DISCUSSION THREAD

**Discussion by IanNorris** [simple-login/app#880] [Feature Requests]
I absolutely LOVE the transfer alias feature SimpleLogin provides. This let me transfer the alias for my wife's main email address on our domain to her, so she can control it, and she's been giving the service a go on her own as well. What'd be even better though is if I could transfer a subdomain of the main domain to her, or even better let her create aliases on the primary domain too as a secondary owner.

She seems quite content to use her own subdomain on the public aliases (another great feature), but one of the main draws of using SimpleLogin for me is having ownership of the domains we use so we're not locked to one provider.

**Comment by IanNorris** [comment]
Based on https://github.com/simple-login/app/discussions/758#discussioncomment-2033835 by @r1ch, I think this could be a partial solution to the above, although not quite as neat, and would leave the aliases in my control rather than my wife's (meaning she can't block addresses etc).

**Reply by nguyenkims** [reply]
She can still block sender or disable alias via the one-click unsubscribe https://simplelogin.io/docs/getting-started/one-click-unsubscribe/ but this is indeed not as convenient as having her own SL account.
